src/cdk/overlay/overlay-ref.ts

```diff
@@ -565,9 +577,10 @@ export class OverlayRef implements PortalOutlet {
 
   /** Removes a backdrop element from the DOM. */
   private _disposeBackdrop(backdrop: HTMLElement | null) {
+    this._cleanupBackdropClick?.();
+    this._cleanupBackdropClick?.();
```

Comment:
should be `_cleanupBackdropTransitionEnd`?

Reply:
Ah good catch.